src/View/SSViewer_Scope.php

```diff
@@ -368,4 +376,15 @@ protected function getUpIndex()
     {
         return $this->upIndex;
     }
+
+    private function convertPrimitiveToDBField(bool|string|float|int|null $value): DBField
+    {
+        return match (gettype($value)) {
+            'boolean' => DBBoolean::create()->setValue($value),
+            'string' => DBText::create()->setValue($value),
+            'double' => DBFloat::create()->setValue($value),
+            'integer' => DBInt::create()->setValue($value),
+            'NULL' => DBText::create()->setValue('NULL')
+        };
+    }
```

Comment:
```suggestion
    private function convertPrimitiveToDBField(bool|string|float|int $value): DBField
    {
        return match (gettype($value)) {
            'boolean' => DBBoolean::create()->setValue($value),
            'string' => DBText::create()->setValue($value),
            'double' => DBFloat::create()->setValue($value),
            'integer' => DBInt::create()->setValue($value)
        };
    }
```
`null` is already correctly handled - it's an empty value and shouldn't be rendered as though it were a string.

Reply:
Done